tests/integration/test_interface.py

```diff
@@ -23,17 +23,19 @@
 
 
 @pytest.mark.parametrize(
-    "data_path, noising_function",
+    "data_path, noising_function, skip",
```

Comment:
The skip parameter will go away once you've built up the config for all forms?

Reply:
Correct, just did that as a temporary helper